### DISCUSSION TITLE
How should I handle these Isoforms Classification?

### DISCUSSION THREAD

**Discussion by bxZhang996** [ConesaLab/SQANTI3#325] [Q&A] [answered]
Dear SQANTI3 Development Team,

Thank you very much for developing SQANTI3, a truly comprehensive and powerful software tool.

However, I encountered an issue while running SQANTI3. Initially, I used TransDecoder on my transcripts and filtered out those without CDS using the Pfam and UniProt-SwissProt databases. I then ran SQANTI3 on this filtered set of transcripts.

![image](https://github.com/user-attachments/assets/d6ea5bfa-cd09-43cf-a919-e2f17cb2317f)

The run was successful, but SQANTI3 flagged some of my transcripts as "Intergenic.". Given that all the transcripts I retained after filtering with the Pfam and SwissProt databases should be coding transcripts, this seems contradictory.

In this situation, should I manually modify the "Intergenic" category to "NNC or other" since these transcripts should theoretically not be non-coding?

I appreciate your help and look forward to your guidance.

Sincerely,
xuan

**Comment by carolinamonzo** [comment, marked as answer]
Hi Xuan, 
I recommend you check the GTF you inputted into SQANTI3. Transcripts classified as intergenic don't have a gene assigned in the associated_gene column. They are intergenic, not any other category, you shouldnt rename them as anything else.
Best,
Carolina.
